tree/tree/inc/TChain.h

```diff
@@ -66,8 +67,8 @@ class TChain : public TTree {
    static constexpr auto kBigNumber = TTree::kMaxEntries;
 
 public:
-   TChain();
-   TChain(const char* name, const char* title = "");
+   TChain(bool locallyManaged = false);
+   TChain(const char *name, const char *title = "", bool locallyManaged = false);
```

Comment:
Can these take an enum class instead of a bool, to make it less likely that they are picked up by users by mistake?